node/prisma/README.md

```diff
@@ -0,0 +1,25 @@
+# Prerequisites
+1. Node/Npm
+2. YugabyteDB 
+
+# Build and run
+
+Install depedencies by running :
+```
+$ cd node/prisma && npm install
+```
+
+Specify the `DATABASE_URL` in the env file by the following command: 
+```
+echo 'DATABASE_URL="postgresql://<user>:<password>@<host>:<port>/<db_name>"' > .env 
+```
+
+Create the tables in the YugabyteDB by applying the migration for the data models in the file `prisma/schema.prisma` using the following command and generate the `PrismaClient`: 
+```
+prisma migrate dev --name first_migration
```

Comment:
Do we need to install prisma explicitly? If so, please add a note for the same (and how to do it).

Reply:
Yes, we are doing `npm install` which will install all the dependencies which are there in `package.json` file and `prisma` is one of those.

Reply:
Ok, I followed the steps above and still could not see prisma command. Am I missing something?